src/CargoSubsystem.cpp

```diff
@@ -38,12 +38,12 @@ void CargoSubsystem::teleop(){
     }
 }
 
-
+//spins wheels to intake the cargo
 void CargoSubsystem::SetIntake(){
     m_rightSideIntake.Set(ControlMode::PercentOutput, 0.2);
     m_leftSideIntake.Set(ControlMode::PercentOutput, 0.2);
 }
-
+//returns true if the intake 
 bool CargoSubsystem::IsIntaking(){
     if(m_cargoLimitSwitch.Get()){
```

Comment:
You could still be intaking if the limit switch is pressed, so you may want to look at the motor speeds instead of what the limit switch is returning. You could make a separate function for whether or not the limit switch is pressed